src/twisted/web/_http2.py

```diff
@@ -103,11 +103,23 @@ class H2Connection(Protocol, TimeoutMixin):
     @ivar _sender: A handle to the data-sending loop, allowing it to be
         terminated if needed.
     @type _sender: L{twisted.internet.task.LoopingCall}
+
+    @ivar abortTimeout: The number of seconds to wait after we attempt to shut
```

Comment:
`timeOut` is a public attribute, and is in fact set by the `Site` object, so I was being consistent with that. This does seem very much like the kind of thing users may want to change.